### PR TITLE
Implement Add creatEllipse mode #54

### PR DESCRIPTION
**Pull request by sivako**

### CODE REVIEW

File: examples/deck/example.js [hunk 1]
```diff
@@ -167,6 +167,7 @@ export default class Example extends Component<
               <option value="drawPolygon">drawPolygon</option>
               <option value="drawRectangle">drawRectangle</option>
               <option value="drawCircle">drawCircle</option>
+              <option value="drawEllipse">drawEllipse</option>
```
Comment (georgios-uber) [inline examples/deck/example.js]: rename to `drawEllipseByBoundingBox`

File: package.json [hunk 2]
```diff
@@ -50,9 +50,11 @@
     "@turf/bbox": ">=4.0.0",
     "@turf/bbox-polygon": ">=4.0.0",
     "@turf/buffer": ">=4.0.0",
+    "@turf/center": "^6.0.1",
```
Comment (georgios-uber) [inline package.json]: Please change spec to `>=4.0.0`

File: examples/deck/example.js (hunk 1 above)
Comment (sivako) [reply examples/deck/example.js]: updated

File: package.json (hunk 2 above)
Comment (sivako) [reply package.json]: updated

**Comment by sivako** [comment]
Updated Documentation.  Thanks @supersonicclay @georgios-uber

**Pull request merged**